lib/stm32wba/hci/README

```diff
@@ -116,6 +116,9 @@ Patch List:
 			  main.h
 			  app_conf.h
 			  scm.c
+			  ll_intf_cmn.h
+			  ll_sys_if.c
+			  linklayer_plat.c
```

Comment:
Please update file list above (line 17) to explain where these files are coming from and update script